### PR TITLE
Adjust vertical centering on links

### PR DESCRIPTION
**Pull request by alexanderankin**
used [this](https://gist.github.com/benweitzel/8621181) as a guide

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -1,14 +1,17 @@
 <header class="site-header">
     
-	<div class="branding">
-		<a href="{{ site.baseurl }}/">
-			<img class="avatar" src="{{ site.baseurl }}/{{ site.theme_settings.avatar }}" alt=""/>
-		</a>
+    <div style="padding: 5px; float: left;">
+        
+        <div class="branding" style="display: inline; float: left;">
```
Comment (meatballhat) [inline _includes/navbar.html]: Can you move inline style like this into `assets/css/main.scss`?

**Review by meatballhat — CHANGES_REQUESTED** [review]
FYI the build error is unrelated, and I'm addressing it in another PR :sweat_smile:

File: _includes/navbar.html (hunk 1 above)
Comment (alexanderankin) [reply _includes/navbar.html]: yep sure thing.

Comment (alexanderankin) [reply _includes/navbar.html]: figured the changes would be more visible here, but they should definately live in the file i just forgot.

Comment (alexanderankin) [reply _includes/navbar.html]: where should i put it? im thinking in navbar but also looking through the other files.

Comment (alexanderankin) [reply _includes/navbar.html]: maybe assets/css/main?

Comment (meatballhat) [reply _includes/navbar.html]: @alexanderankin right, like I mentioned in my first comment :grin:

**Review by meatballhat — APPROVED** [review]

**Pull request merged**